### PR TITLE
fix(atom/tag): fix icon position

### PR DESCRIPTION
**Pull request by AgonisticKatai**
## ATOM/TAG
**TASK**: https://jira.scmspain.com/browse/MTR-38737

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### Description, Motivation and Context
A problem has been detected with the positioning of the icon with respect to the text, causing a misalignment of the icon.

#### BEFORE
image
image

#### AFTER
image
image

### CODE REVIEW

**Review by jordevo — APPROVED** [review]
👏

**Review by rmoralp — APPROVED** [review]

**Review by andresin87 — APPROVED** [review]

File: components/atom/tag/src/index.scss [hunk 1]
```diff
@@ -4,7 +4,10 @@
 
 @mixin icon-atom-tag($type) {
   @include sui-icon--small;
-  display: inline-block;
+  align-content: center;
+  align-items: center;
+  display: inline-flex;
+  justify-content: center;
   line-height: initial;
   vertical-align: middle;
```
Comment (andresin87) [inline components/atom/tag/src/index.scss]: I think this is no-longer needed. (L12)

Comment (AgonisticKatai) [reply components/atom/tag/src/index.scss]: Really, removed!

**Review by turolopezsanabria — COMMENTED** [review]
Hello, the changes don't show as expected in the preview 
(don't match your screenshots)

Screenshot 2021-04-01 at 12 20 21

**Pull request merged**